docs/projMngmnt/POC_Team_Contrib.tex

```diff
@@ -24,170 +24,156 @@
-    Name 4 & Num\\
-    Name 5 & Num\\
+    Total & 3\\
+    Kyle D'Souza & 3\\
+    Mitchell Hynes & 3\\
+    Richard Fan & 3\\
+    Rafeed Iqbal & 3\\
+    Akshit Gulia & 3\\
     \bottomrule
   \end{tabular}
 \end{table}
 
-\wss{If needed, an explanation for the attendance can be provided here.}
-
 \section{Commits}
 
-\wss{For each team member how many commits to the main branch have been made
-  over the time period of interest.  The total is the total number of
-  commits for
-  the entire team since the beginning of the term.  The percentage is the
-percentage of the total commits made by each team member.}
-
 \begin{table}[H]
   \centering
   \begin{tabular}{lll}
     \toprule
     \textbf{Student} & \textbf{Commits} & \textbf{Percent}\\
     \midrule
-    Total & Num & 100\% \\
-    Name 1 & Num & \% \\
-    Name 2 & Num & \% \\
-    Name 3 & Num & \% \\
-    Name 4 & Num & \% \\
-    Name 5 & Num & \% \\
+    Total & 326 & 100\% \\
+    Kyle D'Souza & 79 & 24.2\% \\
+    Mitchell Hynes & 128 & 39.3\% \\
+    Richard Fan & 52 & 16.0\% \\
+    Rafeed Iqbal & 45 & 13.8\% \\
+    Akshit Gulia & 22 & 6.7\% \\
     \bottomrule
   \end{tabular}
 \end{table}
 
-\wss{If needed, an explanation for the counts can be provided here.  For
-  instance, if a team member has more commits to unmerged branches,
-  these numbers
-  can be provided here.  If multiple people contribute to a commit,
-  git allows for
-multi-author commits.}
+The numbers found here come from the contributor insights on the
+GitHub repository.
 
 \section{Issue Tracker}
 
-\wss{For each team member how many issues have they authored (including open and
-  closed issues (O+C)) and how many have they been assigned (only
-    counting closed
-issues (C only)) over the time period of interest.}
-
 \begin{table}[H]
   \centering
   \begin{tabular}{lll}
     \toprule
     \textbf{Student} & \textbf{Authored (O+C)} & \textbf{Assigned (C only)}\\
     \midrule
-    Name 1 & Num & Num \\
-    Name 2 & Num & Num \\
-    Name 3 & Num & Num \\
-    Name 4 & Num & Num \\
-    Name 5 & Num & Num \\
+    Kyle D'Souza & 35 & 25 \\
+    Mitchell Hynes & 77 & 29 \\
+    Richard Fan & 6 & 20 \\
+    Rafeed Iqbal & 1 & 19 \\
+    Akshit Gulia & 3 & 17 \\
     \bottomrule
   \end{tabular}
 \end{table}
 
-\wss{If needed, an explanation for the counts can be provided here.}
+In terms of issues authored the reason it is so imbalanced is because
+generally when assigning work for a deliverable we will all have a
+call and Mitchell will create issues and assign them to people (but
+  we are all agreeing on whats being created and who its being assigned
+to). Recently Kyle did this as well which is why his issues created
+is also very high. To our knowledge issues can't be co-authored, so
+it leads to that disproportionate issue creation statistic.
 
 \section{CICD}
 
-\wss{Say how CICD will be used in your project}
-
-\wss{If your team has additional metrics of productivity, please feel free to
```

Comment:
PR reviews would be a nice metric to include, but idk how to find numbers on them, so I didn't mention it. If anyone else does know though, then maybe we could include that

Reply:
there's this query you can use `is:pr reviewed-by:<username>`, but it also includes comments on your own PR. So like this PR is considered as reviewed by me

Reply:
is:pr -author:<username> reviewed-by:<username> would get u the metrics u want, '-' removes matching entries.

Reviews

- Rafeed : 5 (including this one)
- Kyle : 69
- Akshit : 1
- Richard : 49
- Mitchell : 8